docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+        data, etc. and it also improves security.</para>
+
+        <sect4 id="Taking_adv_of_Container">
+          <title>data</title>
+
+          <para>General data files generated by HPCC are stored stored in
+          data. For Thor, storage costs are likely to be significant.
+          Sequential access speed is important, but random access is much less
+          so. For ROXIE, speed of random access is likely to be most
+          important.</para>
+        </sect4>
+
+        <sect4>
+          <title>lz</title>
+
+          <para>LZ or landing zone data. This is where we would put raw data
+          coming into the system. A landing zone where external users can read
+          and write files. HPCC Systems can import from or export files to a
+          landing zone. Typically performance is less of an issue, it could be
+          blob/s3 bucket storage, accessed either directly or via an NFS
+          mount.</para>
+        </sect4>
+
+        <sect4>
+          <title>dali</title>
+
+          <para>The location of the dali metadata store, which needs to
+          support fast random access.</para>
+        </sect4>
+
+        <sect4>
+          <title>dll</title>
+
+          <para>Where the compiled ECL queries are stored. The storage needs
+          to allow shared objects to be directly loaded from it
+          efficiently.</para>
+
+          <para>If you wanted both Dali and dll data on the same plane, it is
+          possible to use the same prefix for both subpath properties. You
+          have to point to same persistent volume for both, but continue to
+          point to different subdirectories.</para>
+        </sect4>
+
+        <sect4>
+          <title>sasha</title>
+
+          <para>This is the location where archived workunits, etc are stored
+          and it is typically less speed critical, requiring lower storage
+          costs.</para>
+        </sect4>
+
+        <sect4>
+          <title>spill</title>
+
+          <para>An optional category where the spill files from Thor are
+          written. Local NVMe disks are potentially a good choice for
+          this.</para>
+        </sect4>
+
+        <sect4>
+          <title>temp</title>
+
+          <para>An optional category where temp files can be written
+          to.</para>
+        </sect4>
+      </sect3>
+
+      <sect3 id="CYML_BareMEtalStorage">
+        <title>Bare Metal Storage</title>
+
+        <para>There are two aspects to using bare metal storage in the
+        Kubernetes system. The first is the <emphasis>hostGroups</emphasis>
+        entry in the storage section which provides named lists of hosts. The
+        hostGroups entries can take one of two forms. This is the most common
+        form, and directly associates a list of host names with a name:</para>
+
+        <programlisting>storage: 
+  hostGroups: 
+  - name: "The name of the host group process" 
```

Comment:
Do you have a convention for showing values that are to be replaced?
E.g. angle brackets?
- name: "<The name of the host group>"